src/parameterizations/vertical/MOM_opacity.F90

```diff
@@ -40,7 +40,19 @@ module MOM_opacity
   real :: PenSW_flux_absorb !< A heat flux that is small enough to be completely absorbed in the next
                         !! sufficiently thick layer [C H T-1 ~> degC m s-1 or degC kg m-2 s-1].
   real :: PenSW_absorb_Invlen !< The inverse of the thickness that is used to absorb the remaining
-                        !! shortwave heat flux when it drops below PEN_SW_FLUX_ABSORB [H ~> m or kg m-2].
+  !! shortwave heat flux when it drops below PEN_SW_FLUX_ABSORB [H ~> m or kg m-2].
+
+  !! Lookup tables for Ohlmann solar penetration scheme
+  !! These would naturally exist as private module variables but that is prohibited in MOM6
+  real :: dlog10chl           !< Chl increment within lookup table
```

Comment:
Please add a description of the units of dlog10chl and the next 7 variables to the units describing them.

Reply:
Addressed in [f0e2500](https://github.com/NCAR/MOM6/pull/344/commits/f0e25007c4bbf4a80af6be0f149875320cce98af)